core/src/main/java/org/pentaho/platform/scheduler2/quartz/QuartzScheduler.java

```diff
@@ -1029,14 +1080,14 @@ private static List<ITimeRecurrence> parseDayOfWeekRecurrences( String cronExpre
       }
     } else {
       throw new IllegalArgumentException( Messages.getInstance().getErrorString(
-        "ComplexJobTrigger.ERROR_0001_InvalidCronExpression" ) );
+        COMPLEX_JOB_TRIGGER_ERROR_0001_INVALID_CRON_EXPRESSION ) );
     }
     return dayOfWeekRecurrence;
   }
 
   private static List<ITimeRecurrence> parseRecurrence( String cronExpression, int tokenIndex ) {
     List<ITimeRecurrence> timeRecurrence = new ArrayList<>();
-    String delims = "[ ]+";
+    String delims = " +";
```

Comment:
The former use of square brackets left no doubt that there was a space there...
By removing them from the pattern, one gets the same result, but someone in the future may not see the space as significant... maybe add a comment here?